src/lib.rs

```diff
@@ -642,6 +642,20 @@ fn start_tag<'io, 'c, 'l, W: Write>(
                     }
                 }
             }
+            #[cfg(feature = "kitty")]
+            ImageCapability::Kitty(ref kitty) => {
+                let access = ctx.resources.resource_access;
+                if let Some(url) = ctx
+                    .resources
+                    .resolve_reference(&link)
+                    .filter(|url| access.permits(url))
```

Comment:
That's the 3rd block which has this piece of code; it's duplicated in every image block here :(

I'll need to make a mental note to refactor this a little after this pull request is merged :)

No need for you to take any action here though :)  Let's merge this as is :)

Reply:
Good point.